scripts/us_census/geojsons_low_res/plotter.py

```diff
@@ -26,24 +26,21 @@
 matplotlib.use('TkAgg')
 import matplotlib.pyplot as plt
 
-FLAGS = flags.FLAGS
-flags.DEFINE_string('original_path',
-                    default=None,
-                    help='Path to original geojson to be compared.')
-flags.DEFINE_string('simplified_path',
-                    default=None,
-                    help='Path to simplified geojson to be compared.')
-flags.mark_flag_as_required('original_path')
-flags.mark_flag_as_required('simplified_path')
-
 _, (ax1, ax2) = plt.subplots(ncols=2, sharex=True, sharey=True)
 
 
 def compare_plots(geojson1, geojson2, show=True):
-    f1 = geojson1.plot(ax=ax1)
-    f2 = geojson2.plot(ax=ax2)
-    f1.set_title('Original.')
-    f2.set_title('Simplified.')
+    if show:
+        _, (new_ax1, new_ax2) = plt.subplots(ncols=2, sharex=True, sharey=True)
+        f1 = geojson1.plot(ax=new_ax1)
+        f2 = geojson2.plot(ax=new_ax2)
+        f1.set_title('Original.')
+        f2.set_title('Simplified.')
+    else:
+        f1 = geojson1.plot(ax=ax1)
+        f2 = geojson2.plot(ax=ax2)
```

Comment:
How are `ax1` and `ax2` defined?  Are they global variables?

Reply:
They are global variables yes. This is needed in order to have multiple geographies drawn in one plot.